### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by cindy1u0**
let me know if i have typos or if you have other suggestions on what else to put

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,28 @@
...
+<a href="#"><img src="https://cdn.rawgit.com/aleen42/badges/master/src/react.svg" alt="React Badge"></a> <a href="#"><img src="https://cdn.rawgit.com/aleen42/badges/master/src/typescript.svg" alt="Typescript Badge"></a>
+
+[vocabbuddy.now.sh](vocabbuddy.now.sh)
+## Introduction
+Vocab buddy is a web application that identities challenging words and creates corresponding learning interventions to young children expand their vocabulary development. It consists of 8 sessions of assessments and interventions to best track children's progress through out study. The project is developed with React and Typescript, as well as firebase to store research data.
+
+The application is currently developed and maintained by [Sandbox](https://www.sandboxnu.com), Northeastern's student-led software consultancy.
+
+### Assessments
+Assessments are the quizzes/tests that a child will take before a set of interventions. They identity the skill gaps and chidlren' vocabulary knowledge. Each assessment consists of more than 10 wrods, where a child needs to select the best picture that matches the given word.
```
Comment (jrosen081) [inline README.md]:
```suggestion
Assessments are the quizzes/tests that a child will take before a set of interventions. They identity the skill gaps and children' vocabulary knowledge. Each assessment consists of more than 10 words, where a child needs to select the best picture that matches the given word.
```

**Review by jrosen081 — COMMENTED** [review]
I think adding something about how we use firebase as our backend and react-redux-saga for the frontend?

**Comment by cindy1u0** [comment]
> I think adding something about how we use firebase as our backend and react-redux-saga for the frontend?

I think i  did mention firebase is our backend but i will change the frontend part

File: README.md [hunk 2]
```diff
@@ -1 +1,28 @@
-# vocab-buddy
\ No newline at end of file
+# vocab-buddy 
+<a href="#"><img src="https://cdn.rawgit.com/aleen42/badges/master/src/react.svg" alt="React Badge"></a> <a href="#"><img src="https://cdn.rawgit.com/aleen42/badges/master/src/typescript.svg" alt="Typescript Badge"></a>
+
+[vocabbuddy.now.sh](vocabbuddy.now.sh)
+## Introduction
+Vocab buddy is a web application that identities challenging words and creates corresponding learning interventions to young children expand their vocabulary development. It consists of 8 sessions of assessments and interventions to best track children's progress through out study. The project is developed with React, Redux sagas as frontend, as well as firebase as backend to store research data.
```
Comment (jrosen081) [inline README.md]:
```suggestion
Vocab buddy is a web application that identities challenging words and creates corresponding learning interventions to young children expand their vocabulary development. It consists of 8 sessions of assessments and interventions to best track children's progress through out study. The project is developed with React, Redux Saga as frontend, as well as firebase as backend to store research data.
```

**Review by jrosen081 — APPROVED** [review]

**Pull request merged**